src/media-query-tracker.js

```diff
@@ -30,7 +30,7 @@ const getSizes = (data) => {
 		data.innerHeight = global.innerHeight
 	}
 }
-const makeOnResize = dispatch => () => {
+const makeOnResize = dispatch => {
```

Comment:
This won't actually work, it needs to be a function that is called when the resize handler is called, and therefore it needs access to dispatch. It's also not needed to make it work :)